_data/accessibility-tests/search.yml

```
@@ -0,0 +1,175 @@
+    test_status: pass
+    test_type: general
+    version_tested: 3.8.0
+    wcag_criterion: 2.5.5
+  - summary: Page context does not change before search submission.
+    summary_additional: |
+      Unless notified beforehand, page content stays the same until you submit the search. If search results are opened within a new window, an advanced warning should be given alongside search instructions in the `aria-label` or button text.
+    test_status: conditional
+    test_type: screen_reader
+    version_tested: 3.8.0
+    wcag_criterion: 3.2.2
+  - summary: The site search feature appears in the same location on every page of the website.
+    summary_additional: |
+      When you navigate to different pages on the website, the search feature is always in the same place.
+    test_status: conditional
+    test_type: general
+    version_tested: 3.8.0
+    wcag_criterion: 3.2.3
+  - summary: Magnifying glass icon is only used to indicate "search" in every instance on the site.
+    summary_additional: |
+      When you see or tab to the magnifying glass icon anywhere on the site, it will always and only be 
+      used to indicate search.
+    test_status: conditional
+    test_type: general
+    version_tested: 3.8.0
+    wcag_criterion: 3.2.4
+  - summary: Search offers visual and auditory instructions.
+    summary_additional: |
+      When you use the search component, you hear details about the purpose of the input with a screen reader or see them on the page. 
+      The instructions are specific to the search's intent (e.g., "search this site", "search this directory").
+      Instructions may be specific depending on intent (e.g., search this site, search this directory).
+    test_status: conditional
+    test_type: general
+    version_tested: 3.8.0
+    wcag_criterion: 3.3.2
+  - summary: Search instructions are persistent.
+    summary_additional: |
+      When using the search feature, the instructions remain on screen and do not disappear from view.
+    test_status: conditional
+    test_type: general
+    version_tested: 3.8.0
+    wcag_criterion: 3.3.2
+# Zoom/screen magnification tests
+  - summary: Using zoom does not obstruct the text field or search button.
+    summary_additional: |
+      When you zoom to 200%, you can see the text field and button. They do not overlap with other content, and no scrolling is needed.
+    test_status: pass
+    test_type: zoom
+    version_tested: 3.8.0
+    wcag_criterion: 1.4.10
+# Keyboard navigation tests
+  - summary: A mouse is not required to perform actions in the search component. 
+    summary_additional: |
+      When you use a keyboard to tab into the search component, you can type, delete, select and deselect the search 
+      button, and perform the search.
+    test_status: exception
+    status_details: | 
+      Some browsers show an "x" that lets users delete text in the search field. This is not a result of USWDS; it is built in browser behavior. 
```

@sarah-sch @finekatie 
Content folks please correct me, but I believe this needs a hyphen? English is hard.
```suggestion
      Some browsers show an "x" that lets users delete text in the search field. This is not a result of USWDS; it is built-in browser behavior. 
```

---

Good catch! Looking into it it looks like this is correct. Ok if I go ahead and commit?

---

Go for it!